packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -839,7 +839,7 @@ PageHeader.propTypes = {
    * It is used in an overflow menu when there is insufficient space to display all breadcrumbs inline.
    */
   breadcrumbOverflowAriaLabel: PropTypes.string.isRequired.if(
-    ({ breadcrumbs }) => breadcrumbs && breadcrumbs.length > 0
+    ({ breadcrumbs }) => breadcrumbs && breadcrumbs.length > 1 // always shows the title
```

Comment:
I don't think I get this. If the 'breadcrumbs' prop had one or more items in it, we were already moving the title up into the breadcrumb bar on scroll, so the behaviour in that case hasn't changed with this PR. Why does the logic for when we require an aria label for the breadcrumb overflow button change?

Reply:
If anything, does the fact that we might move the title up into the breadcrumb bar on scroll even when there aren't any other breadcrumbs mean that we need a breadcrumb overflow aria-label all the time and not just when there are breadcrumbs?